dumper/app/src/test/java/com/google/edwmigration/dumper/application/dumper/connector/teradata/TeradataLogsConnectorTest.java

```diff
@@ -62,148 +67,174 @@
-                jdbcTemplate.execute(new ConnectionCallback<Void>() {
-                    @Override
-                    public Void doInConnection(Connection connection) throws SQLException, DataAccessException {
-                        PreparedStatement dbqlogtbl = connection.prepareStatement("insert into DBC.DBQLogTbl (\"procid\", \"collecttimestamp\", \"username\", \"errorcode\", \"querytext\", \"queryid\") values (?, ?, ?, ?, ?, ?)");
-                        PreparedStatement dbqlsqltbl = connection.prepareStatement("insert into DBC.DBQLSqlTbl (\"procid\", \"collecttimestamp\", \"queryid\", \"sqlrowno\", \"sqltextinfo\") values (?, ?, ?, ?, ?)");
-
-                        connection.setAutoCommit(false);
-                        long startTime = System.currentTimeMillis();
-                        final int errorCode = 0; // entries with non-zero codes won't get dumped
-                        for (int i_log = 0; i_log < N_QUERY_LOGS; i_log++) {
-                            int procId = i_log % 10;
-                            Timestamp collectTimeStamp = new Timestamp(startTime - i_log); // decrement by one millisecond
-                            final String fullQueryText = "select " + Strings.repeat("'" + i_log + "', ", 50) + "'end';";
-                            final String abbrevQueryText = fullQueryText.substring(0, 100);  // not actual TD behavior, but conveys the point
-                            exec(dbqlogtbl, procId, collectTimeStamp, newUserName(i_log), errorCode, abbrevQueryText, i_log);
-                            exec(dbqlsqltbl, procId, collectTimeStamp, i_log, 0, fullQueryText);
-                            monitor.count();
-                        }
-                        connection.commit();
-
-                        return null;
-                    }
-                });
-            }
-        }
-
-        File outputFile = TestUtils.newOutputFile("compilerworks-teradata-logs-synthetic.zip");
-        FileSystemUtils.deleteRecursively(outputFile);
-
-        List<String> args = Arrays.asList(
-                "--connector", connector.getName(),
-                "--output", outputFile.getAbsolutePath(),
-                "--jdbcDriver", org.postgresql.Driver.class.getName(),
-                "--url", "jdbc:postgresql:cw",
-                "--user", "cw",
-                "--password", "password",
-                "--query-log-days", "1"
-        );
-
-        // Rewrite validity queries from Teradata (SELECT TOP 1 ...) to PostgreSQL:
-        TeradataLogsConnector.TeradataLogsJdbcTask.EXPRESSION_VALIDITY_QUERY = "SELECT %s FROM %s FETCH FIRST 1 ROW ONLY";
-
-        MetadataDumper dumper = new MetadataDumper();
-        dumper.run(args.toArray(ArrayUtils.EMPTY_STRING_ARRAY));
-
-        // TODO: Use ZipValidator to assert that all N_QUERY_LOGS entries are present.
+  }
+
+  @Test
+  public void testConnectorSynthetic() throws Exception {
```

Comment:
I haven't checked exactly, but it looks like this test hasn't changed. If it's only formatting that was changed, I would suggest to move the formatting changes to a separate PR. Similarly for other cases, where only formatting was changed.